ibm_catalog.json

```diff
@@ -75,6 +79,10 @@
                 {
                   "title": "Creates Key Rings and Keys",
                   "description": "Creates Key Rings and Keys in an existing KMS instance."
+                },
+                {
+                  "title": "Collect events which failed delivery",
+                  "description": "Option to connect Cloud Object Storage Services instance to Event Notifications instance to collect the events which failed delivery."
```

Comment:
maybe make it clear the solution supports creating the COS instance and bucket, or using an existing bucket